packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1277,56 +1277,57 @@ export class StripeHelper {
   }
 
   /**
-   * Return a list of skus for a given product.
+   * Return a list of skus for a given price.
    */
-  productToPlaySkus(product: AbbrevProduct) {
-    const productSkus =
-      product.product_metadata[STRIPE_PRODUCT_METADATA.PLAY_SKU_IDS] || '';
-    return productSkus
+  priceToPlaySkus(price: AbbrevPlan) {
+    const priceSkus =
+      price.plan_metadata?.[STRIPE_PRICE_METADATA.PLAY_SKU_IDS] || '';
+    return priceSkus
```

Comment:
Where we check for the SKU ids is probably the most confusing part of this PR to me.

IIRC
 - there's no process preventing product managers from adding the SKU ids to a product and/or the product's prices
 - we currently do have these ids on at least one product

Shouldn't we check the "merged" metadata of a plan and its product for the SKU ids?

Reply:
Oh, that's correct, this does need to be merged just like the others.

Reply:
Ah, actually, this works slightly differently. We do merge them, this code is to pull the SKU from the price. If we looked at the product, then *all the prices for this product* would be considered subscribed prices.